src/app/access-requests/features/access-request-field-edit/access-request-field-edit.component.html

```diff
@@ -2,81 +2,81 @@
+            <mat-form-field class="">
+              <mat-datepicker-toggle
+                matIconSuffix
+                class="float-end"
+                [for]="datepicker"
+              ></mat-datepicker-toggle>
+              <input
                 matInput
+                [matDatepicker]="datepicker"
                 [(ngModel)]="field"
-                (ngModelChange)="changed()"
-                class="min-h-20"
-                [rows]="rows()"
-              ></textarea>
-            } @else {
-              <input matInput [(ngModel)]="field" (ngModelChange)="changed()" />
-            }
-          </mat-form-field>
-        }
-        <mat-chip
-          class="smaller-chip interactive-chip cancel-edit-button mx-2"
-          (click)="cancel()"
-        >
-          <mat-icon>close</mat-icon>
-        </mat-chip>
-        <mat-chip
-          class="smaller-chip interactive-chip save-edit-button"
-          (click)="save()"
-        >
-          <mat-icon>check</mat-icon>
-        </mat-chip>
-      } @else {
-        <span class="grow">
-          @if (ticketUrl()) {
-            <a [href]="ticketUrl()" target="_blank" tabindex="-1"
-              >{{ field()
-              }}<mat-icon
-                role="img"
-                aria-hidden="true"
-                class="mat-icon material-symbols-outlined mat-icon-no-color no-translate h-4 w-4"
-                data-mat-icon-type="font"
-                >open_in_new</mat-icon
-              ></a
-            >
+                (dateChange)="onDateSelected($event)"
+                class="pr-8"
+              />
+
+              <mat-datepicker #datepicker></mat-datepicker>
+            </mat-form-field>
           } @else {
-            @if (isDate()) {
-              {{ field() | date: 'yyyy-MM-dd' }}
-            } @else {
-              {{ field() }}
-            }
+            <mat-form-field class="">
+              @if (rows() > 1) {
+                <textarea
+                  matInput
+                  [(ngModel)]="field"
+                  (ngModelChange)="changed()"
+                  [rows]="rows()"
+                ></textarea>
+              } @else {
+                <input matInput [(ngModel)]="field" (ngModelChange)="changed()" />
+              }
+            </mat-form-field>
           }
-        </span>
-        @if (!locked()) {
           <mat-chip
-            (click)="edit()"
-            class="smaller-chip interactive-chip edit-button ms-2"
-            ><mat-icon class="scale-[0.75]">edit</mat-icon>
+            class="smaller-chip interactive-chip cancel-edit-button mx-2"
+            (click)="cancel()"
+          >
+            <mat-icon>close</mat-icon>
+          </mat-chip>
+          <mat-chip
+            class="smaller-chip interactive-chip save-edit-button"
+            (click)="save()"
+          >
+            <mat-icon>check</mat-icon>
           </mat-chip>
+        } @else {
+          <span class="w-[360px] grow">
```

Comment:
```suggestion
          <span class="grow">
```
The fixed-width messes up display in small screens and also with certain screen widths. There's not need for it.